iree/hal/vulkan/vulkan_device.cc

```diff
@@ -695,36 +698,36 @@ StatusOr<ref_ptr<Event>> VulkanDevice::CreateEvent() {
 StatusOr<ref_ptr<Semaphore>> VulkanDevice::CreateSemaphore(
     uint64_t initial_value) {
   IREE_TRACE_SCOPE0("VulkanDevice::CreateSemaphore");
-#if IREE_HAL_VULKAN_EMULATE_TIMELINE_SEMAPHORES
-  return EmulatedTimelineSemaphore::Create(
-      add_ref(logical_device_),
-      // Triggers necessary processing on all queues due to new values gotten
```

Comment:
This documentation is still worth to keep.

Reply:
Thanks, bad merge removed them. Fixed.